resources/design-document.md

```diff
@@ -0,0 +1,223 @@
+# Project Design Doc - Music Bulletin Board Design
+
+## 1. Problem Statement
+
+The Music Bulletin Board is a service allowing musicians to connect with their peers, within similar areas, to find work and post their needs. Users want to have the ability to detail our their 'want' ads with role needs, music genre expected, locations involved, as well as look at and respond to other user's posted ads. 
+
+## 2. Top Questions to Resolve in Review
+1.) How much depth should go into a user's posted ads? Name/description/role/date/salary?
+    - should these be required?
+
+2.) Will the application allow user's that are responding to posted ads to 'deep-dive' into the user posting the ad? - will that be more of a stretch goal and should this remain, at base level, an app for a user to simply post their ad.
+
+3.) Should a filter/search system be put into place from the perspective of a user looking at ads?
+
+4.) Should there be a separate login validation to determine if someone is looking or posting?
+    - should there be a difference? Should there just be a separate page to allow a user to search through existing ads.
+
+5.) Should an ad include tags?
+
+6.) There is still a case to be debated on whether Locations and Venues should be a table rather than an attribute included with the other tables.
+
+
+## 3. Use Cases
+
+U1. As a user, I want to create a new ad.
+
+U2. As a user, I want to view all my posted ad(s)
+
+U3. As a user, I want to update my ad(s) details. ***Can this be broken up into multiple cases - 
```

Comment:
Unless there are some distinct types of updates, I think having a single story to update the ad is fine.